Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
`sudo chown -R $(whoami):admin /usr/local` maybe? It'll do the top directory and everything in it that way, rather than just the top level.